include/crow/websocket.h

```diff
@@ -186,13 +199,29 @@ namespace crow
             /// Send a binary encoded message.
             void send_binary(std::string msg) override
             {
-                send_data(0x2, std::move(msg));
+                int opcode = 0x2;
+#ifdef CROW_ENABLE_COMPRESSION
+                if (compressor_)
+                {
+                    opcode += 0x40;
+                    msg = compressor_->compress(msg);
+                }
+#endif
+                send_data(opcode, std::move(msg));
             }
 
             /// Send a plaintext message.
             void send_text(std::string msg) override
             {
-                send_data(0x1, std::move(msg));
+                int opcode = 0x1;
+#ifdef CROW_ENABLE_COMPRESSION
+                if (compressor_)
+                {
+                    opcode += 0x40;
```

Comment:
Couldn't find where this comes from. May I ask you to send a link to spec? ;)

Reply:
Sure. I'll look for it. Might take some time. I'm currently very busy sadly.